E2E test regression

Fix the regression introduced by 290.
- #290

Now, the E2E tasks fail with this:

```
date: Max 1 argument (see "date --help")
Error: Process completed with exit code 1.
0s
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -318,7 +318,7 @@ jobs:
           ADB="$ANDROID_HOME/platform-tools/adb"
           mkdir -p results
           set -o pipefail
-          LOGCAT_SINCE=$("$ADB" shell date +'%m-%d %H:%M:%S.000')
+          LOGCAT_SINCE=$("$ADB" shell "date '+%m-%d %H:%M:%S.000'")
           ./gradlew connectedE2EAndroidTest \
             -Pe2eClass=com.gb4pc.e2e.PixelCameraOverlayE2ETest \
             | tee >(grep '^##GB4PC_TEST##' > results/e2e-overlay.ndjson) || {
@@ -345,7 +345,7 @@ jobs:
           ADB="$ANDROID_HOME/platform-tools/adb"
           mkdir -p results
           set -o pipefail
-          LOGCAT_SINCE=$("$ADB" shell date +'%m-%d %H:%M:%S.000')
+          LOGCAT_SINCE=$("$ADB" shell "date '+%m-%d %H:%M:%S.000'")
           ./gradlew connectedE2EAndroidTest \
             -Pe2eClass=com.gb4pc.e2e.GalleryButtonVisualE2ETest \
             | tee >(grep '^##GB4PC_TEST##' > results/e2e-gallery.ndjson) || {

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -318,7 +318,7 @@ jobs:
           ADB="$ANDROID_HOME/platform-tools/adb"
           mkdir -p results
           set -o pipefail
-          LOGCAT_SINCE=$("$ADB" shell "date '+%m-%d %H:%M:%S.000'")
+          LOGCAT_SINCE=$("$ADB" shell "date '+%m-%d %H:%M:%S.000'" | tr -d '\r')
           ./gradlew connectedE2EAndroidTest \
             -Pe2eClass=com.gb4pc.e2e.PixelCameraOverlayE2ETest \
             | tee >(grep '^##GB4PC_TEST##' > results/e2e-overlay.ndjson) || {
@@ -345,7 +345,7 @@ jobs:
           ADB="$ANDROID_HOME/platform-tools/adb"
           mkdir -p results
           set -o pipefail
-          LOGCAT_SINCE=$("$ADB" shell "date '+%m-%d %H:%M:%S.000'")
+          LOGCAT_SINCE=$("$ADB" shell "date '+%m-%d %H:%M:%S.000'" | tr -d '\r')
           ./gradlew connectedE2EAndroidTest \
             -Pe2eClass=com.gb4pc.e2e.GalleryButtonVisualE2ETest \
             | tee >(grep '^##GB4PC_TEST##' > results/e2e-gallery.ndjson) || {

